config/equirectangular_gui.json

```diff
@@ -0,0 +1,58 @@
+{
+  "version": 1,
+  "masteraddress": "localhost",
+  "externalcontrolport": 20500,
+  "settings": {
+    "display": {
+      "swapinterval": 0
+    }
+  },
+  "nodes": [
+    {
+      "address": "localhost",
+      "port": 20401,
+      "windows": [
+        {
+          "name": "OpenSpace",
+          "fullscreen": false,
+          "draw2d": false,
+          "stereo": "none",
+          "pos": { "x": 50, "y": 50 },
+          "size": { "x": 1280, "y": 720 },
+          "viewports": [
+            {
+              "tracked": true,
+              "pos": { "x": 0.0, "y": 0.0 },
+              "size": { "x": 1.0, "y": 1.0 },
+              "projection": {
+                "type": "EquirectangularProjection",
+                "quality": "1k"
+              }
+            }
+          ]
+        },
+        {
+          "name": "GUI",
+          "tags": [ "GUI" ],
+          "fullscreen": false,
+          "draw3d": false,
+          "stereo": "none",
+          "pos": { "x": 50, "y": 50 },
+          "size": { "x": 1280, "y": 720 },
+          "viewports": [
+            {
+              "pos": { "x": 0.0, "y": 0.0 },
+              "size": { "x": 1.0, "y": 1.0 }
+            }
+          ]
+        }
+      ]
+    }
+  ],
+  "users": [
+    {
+      "eyeseparation": 0.065,
+      "pos": { "x": 0.0, "y": 0.0, "z": 0.0 }
+    }
+  ]
+}
```

Comment:
Add new line at the end of the file. Same for all json files.

Reply:
Done